test/collector.test.ts

```diff
@@ -11,33 +11,28 @@ import { Wallet } from '@ethersproject/wallet';
 
 use(solidity);
 
-const FIRST_PARTNER_SHARE = 20;
-const SECOND_PARTNER_SHARE = 30;
-const THIRD_PARTNER_SHARE = 40;
-const FOURTH_PARTNER_SHARE = 10;
-
-const buildPartners = (partners: Wallet[], shares?: number[]) => {
-    return [
-        {
-            beneficiary: partners[0].getAddress(),
-            share: shares?.[0] ?? FIRST_PARTNER_SHARE
-        },
-        {
-            beneficiary: partners[1].getAddress(),
-            share: shares?.[1] ?? SECOND_PARTNER_SHARE
-        },
-        {
-            beneficiary: partners[2].getAddress(),
-            share: shares?.[2] ?? THIRD_PARTNER_SHARE
-        },
-        {
-            beneficiary: partners[3].getAddress(),
-            share: shares?.[3] ?? FOURTH_PARTNER_SHARE
-        }
-    ];
+const PARTNER_SHARES = [20, 30, 40, 10];
+const NUMBER_OF_PARTNERS = 4;
```

Comment:
Is there any reason no to do the following?
```suggestion
const NUMBER_OF_PARTNERS = PARTNER_SHARES.length;
```

Reply:
Yes, that's better. Done.